contracts/TokenManager.sol

```diff
@@ -47,25 +52,23 @@ contract TokenManager is Ownable {
     /// @param _chainlinkAddr the address of the token / USD Chainlink data feed
     /// @param _chainlinkDec the number of decimals the Chainlink data feed uses
     function addAcceptedToken(bytes32 _name, address _addr, uint8 _dec, address _chainlinkAddr, uint8 _chainlinkDec) public onlyOwner {
-        tokens[_name] = Token(_addr, _dec, _chainlinkAddr, _chainlinkDec);
-        tokenNames.push(_name);
+        tokens.push(Token(_name, _addr, _dec, _chainlinkAddr, _chainlinkDec));
     }
 
     function deleteTokenName(uint256 index) private {
```

Comment:
I understand the function name was based on previous code but this function is not doing what it is named since it removes at index rather than based on a token name.